BUILD

```diff
@@ -16,3 +16,8 @@ envoy_cc_library(
     srcs = ["version_generated.cc"],
     deps = ["//source/common/common:version_includes"],
 )
+
+config_setting(
+    name = "force_test_link_static",
+    values = {"define": "FORCE_TEST_LINK_STATIC=yes"},
```

Comment:
I think I mostly grok all of this, but, is this "standard" where the option passed on the command is all caps but the option name is lower case? I assume it's case sensitive? Just trying to understand when to use different cases. The docs didn't make case sensitivity immediately clear.